api-audit/src/main/java/com/capitalone/dashboard/evaluator/StaticSecurityAnalysisEvaluator.java

```diff
@@ -64,27 +71,31 @@ private SecurityReviewAuditResponse getStaticSecurityScanResponse(CollectorItem
         }
 
         CodeQuality returnQuality = codeQualities.get(0);
-        securityReviewAuditResponse.setUrl(collectorItem.getOptions().get("reportUrl").toString());
+        securityReviewAuditResponse.setUrl(collectorItem.getOptions().get(STR_REPORT_URL).toString());
         securityReviewAuditResponse.setCodeQuality(returnQuality);
         securityReviewAuditResponse.setLastExecutionTime(returnQuality.getTimestamp());
 
-        returnQuality.getMetrics().forEach(metric -> {
-            if (metric.getName().equalsIgnoreCase("Score")) {
-                Integer value = Integer.parseInt(metric.getValue());
-                if (value == 100) {
-                    securityReviewAuditResponse.addAuditStatus(CodeQualityAuditStatus.STATIC_SECURITY_SCAN_NO_CLOSED_FINDINGS);
-                } else if (value > 0) {
-                    securityReviewAuditResponse.addAuditStatus(CodeQualityAuditStatus.STATIC_SECURITY_SCAN_OK);
-                } else {
-                    securityReviewAuditResponse.addAuditStatus(CodeQualityAuditStatus.STATIC_SECURITY_SCAN_FAIL);
-                }
-            } else if (metric.getName().equalsIgnoreCase("Critical")) {
-                securityReviewAuditResponse.addAuditStatus(CodeQualityAuditStatus.STATIC_SECURITY_SCAN_FOUND_CRITICAL);
-            } else if (metric.getName().equalsIgnoreCase("High")) {
-                securityReviewAuditResponse.addAuditStatus(CodeQualityAuditStatus.STATIC_SECURITY_SCAN_FOUND_HIGH);
+        Set<CodeQualityMetric> metrics = returnQuality.getMetrics();
+        boolean isSecurityCritical = metrics.stream().anyMatch(metric -> metric.getName().equalsIgnoreCase(STR_CRITICAL));
+        boolean isSecurityHigh = metrics.stream().anyMatch(metric -> metric.getName().equalsIgnoreCase(STR_HIGH));
+
+        if (isSecurityCritical){
```

Comment:
can we do this without multiple if else conditions? Too confusing to read

Reply:
these if/else necessary to give priority over crtitical case over others.